src/geocel/Types.hh

```diff
@@ -18,9 +18,15 @@ namespace celeritas
 // TYPE ALIASES
 //---------------------------------------------------------------------------//
 
-//! Fixed-size array for 3D space
+//! Three-dimensional cartesian coordinates
 using Real3 = Array<real_type, 3>;
 
+//! Two-dimensional cartesian coordinates
+using Real2 = Array<real_type, 2>;
+
+//! Two-dimensional extents
+using Size2 = Array<size_type, 2>;
+
```

Comment:
There are still a few to `using ... = Array<real_type, 2>` and `using ... = Array<size_type, 2>` across the codebase if you want to consolidate everything

Reply:
The `Array<real_type, 2>` are all `Intersections`, where 2 is the number of intersections rather than a spatial coordinate—I don't consider it having the same semantics as a Real2. I've updated the Size2 ones though, thanks :)